### DISCUSSION TITLE
Multi page application?

### DISCUSSION THREAD

**Discussion by pizzadox9999** [konsoletyper/teavm#858] [General]
TeaVM is great for single page applications, but what about multi page applications? I have done it with 4 pages and 4 repos. It is somewhat of a clunky process.

**Comment by konsoletyper** [comment]
Yes, TeaVM does not fit well for developing MPAs, but there's nothing I can do with it. Also, I suggest you not to create 4 repos, it's enough to have one repo, even single module/project, but either multiple main methods (and multiple TeaVM runs for each page), or one main method that takes "page name" as a parameter.

**Reply by pizzadox9999** [reply]
Thanks that would ease the development really.
